consensus/consensus.go

```diff
@@ -581,3 +581,15 @@ func (consensus *Consensus) GetPeerFromID(peerID uint32) (p2p.Peer, bool) {
 func (consensus *Consensus) SendMessage(peer p2p.Peer, message []byte) {
 	host.SendMessage(consensus.host, peer, message, nil)
 }
+
+// Populates the common basic fields for all consensus message.
+func (consensus *Consensus) populateBasicFields(message *consensus_proto.Message) {
```

Comment:
IMHO, better name it as populateMessageFields

Reply:
Done.